src/pip/_internal/network/utils.py

```diff
@@ -55,7 +57,7 @@ def raise_for_status(resp: Response) -> None:
 
 
 def response_chunks(
-    response: Response, chunk_size: int = CONTENT_CHUNK_SIZE
+    response: Response, chunk_size: int = DOWNLOAD_CHUNK_SIZE
```

Comment:
Do we still use different chunk sizes after this PR? If not maybe we can remove this argument altogether.